cmd/serve/serve.go

```diff
@@ -0,0 +1,33 @@
+package serve
+
+import (
+	"github.com/sco1237896/sco-backend/pkg/client"
+	"github.com/sco1237896/sco-backend/pkg/server"
+	"github.com/spf13/cobra"
+)
+
+func NewServeCmd() *cobra.Command {
+
+	options := server.Options{
+		Addr: "localhost",
+		Port: 8080,
```

Comment:
nitpick: it is not uncommon to use a single `addr` property, as example https://github.com/kubernetes-sigs/kubebuilder/blob/6bd9e0162727f72eda233f279b13106404ef98da/testdata/project-v4/cmd/main.go#L56-L57

Reply:
https://github.com/sco1237896/sco-backend/pull/1/commits/6ff10bde09356e56bfbaee9f045c3c6a07fa78d0